libp2p/daemon.ss

```diff
@@ -45,5 +58,9 @@
           (delete-file path))
         status))))
 
+(def (stop-the-libp2p-daemon!)
+  (stop-libp2p-daemon!)
+  (current-libp2p-daemon #f))
```

Comment:
Shouldn't `stop-libp2p-daemon!` always call `(current-libp2p-daemon #f)` ? And then this function becomes unnecessary?